rotimi-best/classroomio
# feat: Integrate Sentry to backend

## What does this PR do?



Fixes #256 



## Type of change




## How should this be tested?



- Create your sentry account on sentry.io, add your DSN key to your env file then open your terminal, switch directory to the backend folder then in the server.js file, create a route and throw an error in the route, then run "node server.js", you'll get a mail and you'll see the error in the issues section of your sentry dashboard.

## Checklist



### Required


### Appreciated
